### User manuals and tutorials

1. <a href="https://www.bioconductor.org/packages/release/bioc/vignettes/Prostar/inst/doc/Prostar_UserManual.pdf" target="_blank">Prostar user manual</a> (corresponds to Prostar release 1.14)
2. <a href="https://sites.google.com/site/thomasburgerswebpage/download/Prostar_Tutorial.pdf"  target="_blank">Prostar tutorial</a>  (corresponds to Prostar release 1.10)
3. <a href="https://sites.google.com/site/thomasburgerswebpage/download/tutorial-CP4P-4.pdf" target="_blank">cp4p tutorial</a>
4. <a href="https://sites.google.com/site/thomasburgerswebpage/download/MiMB_Burger_FinalVersion2.pdf" target="_blank">Prostar protein-level protocol</a> (corresponds to Prostar release 1.14)

### Contact

If you need any help, but also if you wish to make comments or suggestions, please contact Samuel Wieczorek or Thomas Burger (firstname.lastname@cea.fr).

### Reference manuals

1. <a href="https://www.bioconductor.org/packages/release/bioc/manuals/Prostar/man/Prostar.pdf" target="_blank">Prostar reference manual</a>
2. <a href="https://www.bioconductor.org/packages/release/bioc/manuals/DAPAR/man/DAPAR.pdf" target="_blank">DAPAR reference manual</a>
3. <a href="https://www.bioconductor.org/packages/release/bioc/html/MSnbase.html" target="_blank">MSnbase package webpage</a>
4. <a href="https://cran.r-project.org/web/packages/cp4p/cp4p.pdf" target="_blank">CP4P reference manual</a>
5. <a href="https://cran.r-project.org/web/packages/imp4p/imp4p.pdf" target="_blank">IMP4P reference manual</a>

### Bibliographical references

#### Our referenced works

1. <a href="http://prabig-prostar.univ-lyon1.fr/Articles/calib-final.pdf" target="_blank">_Q. Giai Gianetto, F. Combes, C. Ramus, C. Bruley, Y. Cout&eacute; and T. Burger_. Calibration Plot for Proteomics (cp4p): A graphical tool to visually check the assumptions underlying FDR control in quantitative experiments. **Proteomics**, 16(1):29-32, 2016.</a>
2. <a href="http://prabig-prostar.univ-lyon1.fr/Articles/natureOfMV-Vsubmited2.pdf" target="_blank">_C. Lazar, L. Gatto, M. Ferro, C. Bruley, T. Burger_. Accounting for the multiple natures of missing values in label-free quantitative proteomics datasets to compare imputation strategies. **Journal of Proteome Research**, 15(4):1116-1125, 2016.</a>
3. <a href="http://prabig-prostar.univ-lyon1.fr/Articles/OnTheMissuseOfFudgeFactorInProteomics_FV.pdf" target="_blank">_Q. Giai Gianetto, Y. Coute, C. Bruley and T. Burger_. Uses and misuses of the fudge factor in quantitative discovery proteomics. **Proteomics**, 16(14):1955-60, 2016.</a>
4. <a href="http://prabig-prostar.univ-lyon1.fr/Articles/prostar.pdf" target="_blank">_S. Wieczorek, F. Combes, C. Lazar, Q. Giai-Gianetto, L. Gatto, A. Dorffer, A.-M. Hesse, Y. Cout&eacute;, M. Ferro, C. Bruley and T. Burger_. DAPAR & ProStaR: software to perform statistical analyses in quantitative discovery proteomics, **Bioinformatics**, 33(1):135-136, 2017</a>
5. <a href="http://prabig-prostar.univ-lyon1.fr/Articles/fdrtuto.pdf" target="_blank">_T. Burger_. Gentle introduction to the statistical foundations of false discovery rate in quantitative proteomics. **Journal of Proteome Research**, 17(1):12-22, 2017.</a>
6. <a href="http://prabig-prostar.univ-lyon1.fr/Articles/revised-biostat-proteom-preprint.pdf" target="_blank">_L. Jacob, F. Combes and T. Burger_. PEPA test : fast and powerful differential analysis from relative quantitative proteomics data using shared peptides. **Biostatistics**, kxy021, 2018.</a>
7. <a href="https://sites.google.com/site/thomasburgerswebpage/download/MiMB_Burger_FinalVersion2.pdf" target="_blank">_S. Wieczorek, F. Combes, H. Borges, T. Burger_. Protein-Level Statistical Analysis of Quantitative Label-Free Proteomics Data with ProStaR. **Proteomics for Biomarker Discovery, V. Brun, Y. Coute (Eds.), Methods in Molecular Biology Series**, Vol. 1959, pp. 225-246, 2019.</a>
8. <a href="https://sites.google.com/site/thomasburgerswebpage/download/LetterSpectralClusteringRevised2.pdf" target="_blank">_H. Borges(+), R. Guibert(+), O. Permiakova(+), T. Burger_. Distinguishing between spectral clustering and cluster analysis of mass spectra.  Journal of Proteome Research, 18 (1), pp 571-573, 2019.</a>
9. <a href="https://sites.google.com/site/thomasburgerswebpage/download/5tips-VF-accepted.pdf" target="_blank">_S. Wieczorek, Q. Giai Gianetto, T. Burger_. Five simple yet essential steps to correctly estimate the rate of false differentially abundant proteins in mass spectrometry analyses.  Journal of Proteomics, Accepted manuscript, In Press, 2019.</a>
10. _Q. Giai Gianetto, S. Wieczorek, Y. Coute and T. Burger_. Multiple peptide-level imputation strategy for mass spectrometry-based proteomic data. (in preparation).

#### Other references

1. _Bolstad BM_ (2017). preprocessCore: A collection of pre-processing functions. **R package** version 1.38.1
2. _Hastie T, Tibshirani R, Narasimhan B and Chu G_ (2017). impute: Imputation for microarray data. **R package** version 1.50.1
3. _Gatto L and Lilley K_ (2012). MSnbase - an R/Bioconductor package for isobaric tagged mass spectrometry data visualization, processing and quantitation. **Bioinformatics**, 28, pp. 288-289.
4. _Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W and Smyth GK_ (2015). Limma powers differential expression analyses for RNA-sequencing and microarray studies. **Nucleic Acids Research**, 43(7), pp. e47.
5. _Cleveland, W. S., & Devlin, S. J._ (1988). Locally weighted regression: an approach to regression analysis by local fitting. **Journal of the American statistical association**, 83(403), 596-610.
6. _Huber, W., Von Heydebreck, A., Sultmann, H., Poustka, A., & Vingron, M._ (2002). Variance stabilization applied to microarray data calibration and to the quantification of differential expression. **Bioinformatics**, 18(suppl_1), S96-S104.


